installer/conf/omsagent.d/oms_audits.xml

```diff
@@ -767,9 +767,13 @@
       remediation="Run the command '/usr/local/bin/azsecd remediate -r disable-rpcbind'. This will disable the rpcbind service."
       ruleId="f4a80328-1d67-45ed-b915-274d2e6c699e">
       <check
-        distro="*"
+        distro="Debian|Ubuntu|Oracle|CentOS&lt;7|RedHat&lt;7|SLES=11"
```

Comment:
@taglines 
Tad, could you please answer on questions regarding distro and versions